superbench/benchmarks/micro_benchmarks/tensorrt_inference_performance.py

```diff
@@ -38,7 +35,6 @@ def __init__(self, name, parameters=''):
             'vgg16',
             'vgg19',
```

Comment:
Do we need to add more?

Reply:
no? can be added in config

Reply:
A little bit confused here because `_pytorch_models` here is a little bit misleading.

Reply:
any suggestions?